### PR TITLE
1. Provide DB schema for events & interest registrations, populate events with entry for PDF CEE 2018. 2. Save interest registration submissions to the DB.

### PR DESCRIPTION
**Pull request by janzankowski**

### CODE REVIEW

File: config/database.php [hunk 1]
```diff
@@ -48,10 +48,10 @@
             'password' => env('DB_PASSWORD', ''),
```
Comment (janzankowski) [inline config/database.php]: For this to work, one needs to set DB_DATABASE, DB_USERNAME, and DB_PASSWORD in .env file.

Obviously, we also need to set up the DB user and schema on the "portals" server, and run the migration.

In MySQL:

- `CREATE USER 'pdfcee'@'localhost' IDENTIFIED BY <<password here>>;`
- `CREATE DATABASE pdfcee CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_520_ci;`
- `GRANT ALL PRIVILEGES ON 'pdfcee'.* TO 'pdfcee'@'localhost';`
- `FLUSH PRIVILEGES;`

In shell:

- `php artisan migrate`

I'm not doing it yet, waiting for your review (plus I don't have the DB credentials).

File: app/Http/Controllers/InterestRegistrationController.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            'organization' => $request->input('organization'),
+            'email' => $request->input('email'),
+            'country' => $request->input('country'),
+            'city' => $request->input('city'),
+            'additional_info' => $request->input('additional-info'),
+            'email_opt_in' => $request->input('email-opt-in', false),
+            'event' => $request->input('event')
+        ]);
+
+        return redirect('/'); // TODO: Thank you page.
```
Comment (danielmacyszyn) [inline app/Http/Controllers/InterestRegistrationController.php]: Let's use [Laravel Flash Data](https://laravel.com/docs/5.5/session#flash-data) for displaying thank you message

File: routes/web.php [hunk 3]
```diff
@@ -14,3 +14,5 @@
 Route::get('/', function () {
     return view('home');
 });
+
+Route::post('/interest-registrations', 'InterestRegistrationController@store');
```
Comment (danielmacyszyn) [inline routes/web.php]: Maybe `Route::post('/', 'InterestRegistrationController@store');` (and setting `action` attribute of the form to `/`) would be nicer?

File: resources/views/home.blade.php [hunk 4]
```diff
@@ -45,36 +45,47 @@
                 <h1>Headline 1</h1>
                 <h2>Headline 2</h2>
             </div>
-            <form class="form-horizontal">
+            <form class="form-horizontal" action="/interest-registrations" method="post">
```
Comment (danielmacyszyn) [inline resources/views/home.blade.php]: See https://github.com/epforgpl/pdfcee/pull/3/files#r140970855

File: database/migrations/2014_10_12_100000_create_password_resets_table.php [hunk 5]
```diff
@@ -1,32 +0,0 @@
-<?php
-
-use Illuminate\Support\Facades\Schema;
-use Illuminate\Database\Schema\Blueprint;
-use Illuminate\Database\Migrations\Migration;
-
-class CreatePasswordResetsTable extends Migration
```
Comment (danielmacyszyn) [inline database/migrations/2014_10_12_100000_create_password_resets_table.php]: What's this for?

File: routes/web.php (hunk 3 above)
Comment (janzankowski) [reply routes/web.php]: Done

File: resources/views/home.blade.php (hunk 4 above)
Comment (janzankowski) [reply resources/views/home.blade.php]: Done

File: app/Http/Controllers/InterestRegistrationController.php (hunk 2 above)
Comment (janzankowski) [reply app/Http/Controllers/InterestRegistrationController.php]: Will do in the next PR.

**Pull request merged**